### ISSUE TITLE
Ctrl+Shift+F10 waits before opening the ToolTip

### ISSUE THREAD

**Issue by SamBent** [dotnet/wpf#4615]
* .NET Core Version: 3.0+
* Windows version: all
* Does the bug reproduce also in WPF for .NET Framework 4.8?: Yes
* Is this bug related specifically to tooling in Visual Studio (e.g. XAML Designer, Code editing, etc...)? no
* Security issues and bugs should be reported privately, learn more via our [responsible disclosure guidelines](https://github.com/dotnet/wpf/blob/master/README.md#reporting-security-issues-and-security-bugs).

 
 
 **Problem description:**
1. Run sample app from #4612
2. Press TAB repeatedly to move focus to the second Image (look for the focus adorner)
3. Press Ctrl+Shift+F10 to toggle the ToolTip
 
 **Actual behavior:** 
Toggle-on shows the ToolTip after a delay (5 sec).
Toggle-off hides the ToolTip immediately.

 **Expected behavior:**
 Both actions occur immediately.

User's explicit request to show/hide the tooltip should be honored immediately.

**Comment by SamBent** [comment]
Fixed by the tooltip work in 6.0-preview7.
